zulip_tools.py

```diff
@@ -9,6 +8,8 @@
 import sys
 import time
 
+from django.utils import timezone
```

Comment:
This script can't import from `django`, since it's part of the zulip installation bootstrapping process.